x/swap/keeper/deposit.go

```diff
@@ -0,0 +1,53 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/kava-labs/kava/x/swap/types"
+)
+
+func (k Keeper) Deposit(ctx sdk.Context, depositor sdk.AccAddress, amountA sdk.Coin, amountB sdk.Coin) error {
```

Comment:
```suggestion
func (k Keeper) Deposit(ctx sdk.Context, depositor sdk.AccAddress, coinA sdk.Coin, coinB sdk.Coin) error {
```

Using the naming convention of 'amount' feels weird because each coin has an `amount` and `denom`

Reply:
👍 didn't like amount either, coin is much clearer